pyproject.toml

```diff
@@ -54,6 +53,8 @@ check-manifest = "*"
 tox = "^3.26"
 flake8 = "==5.0.4"
 black = "^22.10"
+google-api-python-client = "*"
+googleapis-common-protos = "*"
```

Comment:
Do we need both? This looks like a new dependency - was it intended to replace the other Google python api?

Reply:
Looks like we need only googleapis-common-protos

Reply:
from google.api import annotations_pb2